### ISSUE TITLE
`make dist` fails with a read-only GOROOT direcotry

### ISSUE THREAD

**Issue by chengweiv5** [mattermost/platform#669]
Generally, the `GOROOT` directory is read-only for develop user, while mattermost Makefile use `-a` option for `go build/install`, and it will fail with below error.

```
go install runtime: open /usr/local/go/pkg/linux_amd64/runtime.a: permission denied
godep: go exit status 1
Makefile:208: recipe for target 'dist' failed
make: *** [dist] Error 1
```

and seems golang community doesn't goto *fix* this. https://github.com/golang/go/issues/5065

Make GOROOT rw for non-root user seems isn't reasonable, so should we remove `-a` option?

**Comment by apaatsio** [comment]
I'm having this same issue with Arch Linux.

**Comment by crspeller** [comment]
@chengweiv5 Thanks for the report! This has been fixed #694.
